src/components/MiniPersonPhoto.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
```

Comment:
Is there a reason this component is a `div` instead of an `img`? The later is more semantic and accessible. We should throw the `alt` attribute on here too with the person's name.

Reply:
Only because it was how the other cards were laid out and rendered; vue was being pointed to the `background-image` property for the other two person cards so I left it consistent.

Reply:
Ah, I see. Well, let's revisit it later for all 3 card types in the future. I'm noticing that the FullSize card also has a single div with no children for its image. So that should be easily converted to `<img>` as well.